plugins/de.cognicrypt.codegenerator/.classpath

```diff
@@ -38,5 +38,11 @@
 		</attributes>
 	</classpathentry>
 	<classpathentry kind="lib" path="lib/claferchocoig.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">

```

Comment:
@shahrzadav Please remove this code. It causes build path problems due to duplicate JRE entries